docker/reference/normalize.go

```diff
@@ -29,11 +29,18 @@ type normalizedNamed interface {
 // transforming a familiar name from Docker UI to a fully
 // qualified reference. If the value may be an identifier
 // use ParseAnyReference.
-func ParseNormalizedNamed(s string) (Named, error) {
+func ParseNormalizedNamed(sys *types.SystemContext, s string) (Named, error) {
 	if ok := anchoredIdentifierRegexp.MatchString(s); ok {
 		return nil, fmt.Errorf("invalid repository name (%s), cannot specify 64-byte hexadecimal strings", s)
 	}
-	domain, remainder := splitDockerDomain(s)
+	registries, err := GetRegistries(sys)
+	if err != nil {
+		return nil, err
+	}
+	if len(registries) == 0 {
+		return nil, errors.New("no registries specified")
```

Comment:
"no registries specified in /etc/containers/registries.conf" perhaps?

Reply:
The problem is I am not sure if that is the path.